chromium_src/third_party/blink/common/features.cc

```diff
@@ -40,5 +40,9 @@ const base::Feature kFileSystemAccessAPI{"FileSystemAccessAPI",
 const base::Feature kPartitionBlinkMemoryCache{
     "PartitionBlinkMemoryCache", base::FEATURE_DISABLED_BY_DEFAULT};
 
+// Enable WebSockets connection pool limit per eTLD+1 for each renderer.
+const base::Feature kRestrictWebSocketsPool{"RestrictWebSocketsPool",
+                                            base::FEATURE_DISABLED_BY_DEFAULT};
```

Comment:
perhaps we can enable by default, the change doesn't look very dangerous?

Reply:
👌 I'll make it enabled by default and will add a `brave://flag` to be safe.